src/app/space-wizard/components/forge-app-generator/forge-app-generator.component.html

```diff
@@ -0,0 +1,106 @@
+      <h2>{{title}}</h2>
+      <span (click)="workflow.cancel()" class="glyphicon glyphicon-remove"></span>
+    </div>
+  </div>
+</header>
+<section class="container-fluid padding-top-standard-offset">
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="form-horizontal">
+        <div class="form-group" *ngFor="let field of forge.fieldSet">
+          <label class="control-label col-xs-4">{{field.display.label}}</label>
+          <div class="col-xs-8" [ngSwitch]="field.display.inputType">
+            <div *ngSwitchCase="'singleInput'" [ngSwitch]="field.valueType">
+              <input *ngSwitchCase="'string'" name="{{field.name}}" [(ngModel)]="field.value" type="text"
+                     class="form-control" placeholder="{{field.description}}"
+                     [required]="field.display.required" [disabled]="!field.display.enabled"/>
+              <input *ngSwitchCase="'boolean'" type="checkbox" name="{{field.name}}" [(ngModel)]="field.value"
+                     class="form-control" [required]="field.display.required"
+                     [disabled]="!field.display.enabled">
+              <input *ngSwitchCase="'number'" type="number" name="{{field.name}}" [(ngModel)]="field.value"
+                     class="form-control" [required]="field.display.required"
+                     [disabled]="!field.display.enabled">
+            </div>
+            <div *ngSwitchCase="'singleSelection'">
+              <select class="form-control" [(ngModel)]="field.value" name="{{field.name}}"
+                      *ngIf="field.valueType != 'projectType'">
+                <option value="{{option.id}}" *ngFor="let option of field.display.options">
+                  {{option.id}}
+                </option>
+              </select>
+            </div>
+            <div *ngSwitchCase="'multipleSelection'"
+                 style="border:solid 1px #bbb; overflow:hidden;  ; border-radius:4px;">
+              <!--input-->
+              <div class="input-group">
+
+                <div class="input-group-btn">
+                  <div (click)="toggleSelectAll(field)" class="btn btn-default btn-left-multi-select-button">
+
+                    <!--<span [ngClass]="{'glyphicon-check' : field.display.allOptionsSelected,'glyphicon-unchecked' : !field.display.allOptionsSelected}"  class="glyphicon"></span>-->
+                    <span
+                      [ngClass]="{'glyphicon-check' : allOptionsSelected(field),'glyphicon-unchecked' : !allOptionsSelected(field)}"
+                      class="glyphicon"></span>
+                  </div>
+                </div>
+
+                <input class="form-control form-control-filter-input"
+                       (keyup)="filterUnselectedList(field,optionFilter.value)" #optionFilter>
+
+              </div>
+              <!--included list-->
+              <div class="included-list">
+                <div [class.hidden]="!fieldOption.selected" [class.selected]="fieldOption.selected"
```

Comment:
It looks like you might be trying to access fieldOption outside of the ngFor loop that it is declared in. 
And again in the next section.